### ISSUE TITLE
Error installing fbcunn at last step?

### ISSUE THREAD

**Issue by mszlazak** [facebook/fbcunn#64]
I had the latest cuda installed:
cuda-repo-ubuntu1404-7-0-local_7.0-28_amd64.deb
before going through these instruction. Had previous errors with installing Folly etc that was fixed by adding export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/lib

However this second error still occurs when installing fbcunn at the last step:

cd fbcunn && luarocks make rocks/fbcunn-scm-1.rockspec

Here is the end of the terminals output. Lot's of those CMake warnings about removing non-existent dependency files before what I have showing in the snippet below ... if that matters??

CMake Warning at /usr/share/cmake-2.8/Modules/FindCUDA/make2cmake.cmake:66 (message):
Removing non-existent dependency file: generic/THTensorMath.h

CMake Warning at /usr/share/cmake-2.8/Modules/FindCUDA/make2cmake.cmake:66 (message):
Removing non-existent dependency file: generic/THTensorConv.h

CMake Warning at /usr/share/cmake-2.8/Modules/FindCUDA/make2cmake.cmake:66 (message):
Removing non-existent dependency file: generic/THTensorLapack.h

[ 43%] Building NVCC (Device) object CMakeFiles/fbcunnlayers_cuda.dir/src/./fbcunnlayers_cuda_generated_MM.cu.o
Scanning dependencies of target fbcunnlayers_cuda
[ 45%] Building CXX object CMakeFiles/fbcunnlayers_cuda.dir/src/fft/CuFFTConvolution.cpp.o
In file included from /home/mark/nn/fbtorch/fbnn/fbcunn/src/fft/CuFFTConvolution.cpp:14:0:
/home/mark/nn/fbtorch/fbnn/fbcunn/src/fft/Utils.h:5:25: fatal error: thpp/Tensor.h: No such file or directory
#include "thpp/Tensor.h"
^
compilation terminated.
make[2]: *** [CMakeFiles/fbcunnlayers_cuda.dir/src/fft/CuFFTConvolution.cpp.o] Error 1
make[1]: *** [CMakeFiles/fbcunnlayers_cuda.dir/all] Error 2
make: *** [all] Error 2

Error: Build error: Failed installing.

**Comment by feitz** [comment]
I encountered similar errors in the last steps. Anyone can point out where did we go wrong? Thanks!

[ 25%] Building CXX object CMakeFiles/fbcunnlayers_cuda.dir/src/fft/CuFFTConvolution.cpp.o
In file included from /home/fh/fbtorch/fbnn/fbcunn/src/fft/CuFFTConvolution.cpp:14:0:
/home/fh/fbtorch/fbnn/fbcunn/src/fft/Utils.h:5:25: fatal error: thpp/Tensor.h: No such file or directory
 #include "thpp/Tensor.h"
                         ^
compilation terminated.
make[2]: *** [CMakeFiles/fbcunnlayers_cuda.dir/src/fft/CuFFTConvolution.cpp.o] Error 1
make[1]: *** [CMakeFiles/fbcunnlayers_cuda.dir/all] Error 2
make: *** [all] Error 2

Error: Build error: Failed installing.

**Comment by soumith** [comment]
sorry about this. I just fixed the fblualib install script so that this error does not happen now.
I made sure tagged versions of folly and fbthrift are installed, rather than trunk. I tested the install process on a fresh ubuntu 14.04 machine.

You'll have to redo the install process starting from this step:
https://github.com/facebook/fbcunn/blob/master/INSTALL.md#install-folly-fbthrift-thpp-and-fblualib

If you are having installation errors after retrying the install once more (after my fresh fixes), 
I can post the binaries here for Ubuntu 14.04 (they will be specific to ubuntu 14.04).
